### PR TITLE
Fix binder badge

### PR DESCRIPTION
**Pull request by po09i**
The binder badge has been broken. Binder seems to have switched some default behaviour at the beginning of 2022 which coincides with then we observed the problem. This thread discusses it in more detail. The fix specifies in the path to use the old behaviour.

### CODE REVIEW

**Comment by po09i** [comment]
@sandrinebedard Can you test if the badge now works for you

**Review by jcohenadad — COMMENTED** [review]
currently testing-- after 20 minutes it is still loading:
image

**Review by jcohenadad — COMMENTED** [review]
and after 25 min it failed:

image

**Comment by sandrinebedard** [comment]
It loaded in 2 seconds for me!

image

**Comment by jcohenadad** [comment]
ok let me try again

**Comment by jcohenadad** [comment]
> It loaded in 2 seconds for me!

I'm a bit suspicious about the 2 seconds because conda needs to be installed, and then all deps need to be downloaded and installed. Even locally it takes at least 30s or more, no?  Is it possible that it was using a cached version of MyBinder in your browser?

**Comment by sandrinebedard** [comment]
> > It loaded in 2 seconds for me!
> 
> I'm a bit suspicious about the 2 seconds because conda needs to be installed, and then all deps need to be downloaded and installed. Even locally it takes at least 30s or more, no? Is it possible that it was using a cached version of MyBinder in your browser?

That's what I am thinking too! But I never managed to load it before, that's why I am also suspicious ... I am testing it in a private browsing tab too.

**Comment by jcohenadad** [comment]
Another try, resulting in an error after 25 minutes:
image

(also in private browsing tab). 

I suggest asking other students in the lab to run it and get feedback.

**Comment by po09i** [comment]
I also ran into problems... (It hung there for 15 minutes)

Screen Shot 2022-12-01 at 12 01 15 PM

**Comment by sandrinebedard** [comment]
on the private browsing tab, it took 15 mins to load and works now!

**Comment by gaspardcereza** [comment]
I tried it in incognito mode and it only took about 10s. I executed some cells in lab 3 and everything behaves as expected. Maybe binder just got very very good at creating environments quickly, but that would not explain why it fails on your end...

From what I recall, Binder does not always use the same hosting service, maybe that's part of the explanation.

**Comment by po09i** [comment]
I tried a few things, notably installing jupiter lab instead. It seems more reliable now.. I was previously able to replicate Julien's error and now it seems to work pretty reliably when building from scratch, in private windows and in non private windows. When it does not work, I just reload the page and it usually works the 2nd time.

**Comment by po09i** [comment]
Binder seems to have [issues when using an adblocker](https://discourse.jupyter.org/t/mybinder-org-launches-just-hang-are-you-using-an-ad-blocker-ghostery-privacybadger-etc/3511), we might want to suggest to turn it off in the README.

**Comment by jcohenadad** [comment]
we're making progress 🎉   it worked for me (4min of loading):
image

i'm just afraid their servers are unreliable, but who knows... 🤷

**Review by sandrinebedard — COMMENTED** [review]
It worked for after 1min 43 s on a private browser page!

**Review by sandrinebedard — APPROVED** [review]
